ui/src/pages/backoffice/Document/DocumentDetails/components/DocumentMetadata/DocumentCopyrights.js

```diff
@@ -0,0 +1,28 @@
+import { MetadataTable } from '@pages/backoffice';
+import capitalize from 'lodash/capitalize';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export class DocumentCopyrights extends Component {
+  renderCopyrights() {
+    const { copyrights } = this.props.document.metadata.copyrights;
+    const rows = [];
+
+    for (const [key, val] of Object.entries(copyrights)) {
+      rows.push({ name: capitalize(key), value: val });
+    }
+
+    return rows;
+  }
+
+  render() {
+    if (this.props.document.metadata.copyrights) {
```

Comment:
I would do this check in the other function and I would keep `render` as simple as possible.
```js
render() {
  return <MetadataTable rows={this.renderCopyrights()} />;
}
```